src/cephadm/tests/test_cephadm.py

```diff
@@ -305,6 +305,138 @@ def test_last_local_images(self):
         image = cd._filter_last_local_ceph_image(out)
         assert image == 'docker.io/ceph/ceph:v15.2.5'
 
+    def test_parse_ss_lpnt(self):
```

Comment:
I think the the test can be centered in if the port is in the "ss" output,  or not.